From a30a2d89fc3013b89f7872c7ea6e3337dc1d4390 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 1 Jan 2023 13:26:33 +0100
Subject: [PATCH] turris-webapps: update to pre-release 4.0.0

This version of WebApps is not officially tagged in our repository and
it is waiting on review until it will be done, have some testing version
in the distribution

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 web/turris-webapps/turris-webapps/Makefile | 28 +++++++++++-----------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/web/turris-webapps/turris-webapps/Makefile b/web/turris-webapps/turris-webapps/Makefile
index 424b460f89..5ccaac34b7 100644
--- a/web/turris-webapps/turris-webapps/Makefile
+++ b/web/turris-webapps/turris-webapps/Makefile
@@ -8,13 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=turris-webapps
-PKG_VERSION:=3.7.1
+PKG_VERSION:=d05db1e4
 PKG_RELEASE:=$(AUTORELEASE)
 
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/webapps
-PKG_MIRROR_HASH:=67e355dd98edd09df77dfc7c2c41655e0d6b40e89c364ac861d108b2d6b287f6
-PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://gitlab.com/api/v4/projects/42225851/packages/generic/turris-webapps/3.8.3/
+PKG_HASH:=ff257f22427b29c3cf9543f239ae7d12c5feb8c3ca10bf7179e59e70e3f76896
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
 PKG_LICENSE:=GPL-3.0-or-later
@@ -42,21 +41,22 @@ define Package/turris-webapps/install
 
 	$(INSTALL_DIR) $(1)/usr/share/turris-webapps
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/turris-webapps-json-cgi $(1)/usr/share/turris-webapps
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/l10n.json $(1)/usr/share/turris-webapps
 
 	$(INSTALL_DIR) $(1)/www/turris-webapps
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/favicon-*.png $(1)/www/turris-webapps
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/favicon.ico $(1)/www/turris-webapps
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/*.png $(1)/www/turris-webapps
 	$(INSTALL_DATA) $(PKG_BUILD_DIR)/index.html $(1)/www/turris-webapps
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/manifest.json $(1)/www/turris-webapps
 
-	$(INSTALL_DIR) $(1)/www/turris-webapps/css
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/css/*.min.css $(1)/www/turris-webapps/css
+	$(INSTALL_DIR) $(1)/www/turris-webapps/static
 
-	$(INSTALL_DIR) $(1)/www/turris-webapps/js
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/js/*.min.js $(1)/www/turris-webapps/js
-	$(INSTALL_DIR) $(1)/www/webapps-icons
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/icons/turris-dark.svg $(1)/www/webapps-icons
+	$(INSTALL_DIR) $(1)/www/turris-webapps/static/css
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/static/css/* $(1)/www/turris-webapps/static/css
 
+	$(INSTALL_DIR) $(1)/www/turris-webapps/static/js
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/static/js/* $(1)/www/turris-webapps/static/js
+
+	$(INSTALL_DIR) $(1)/www/turris-webapps/static/media/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/static/media/turris-dark.*.svg $(1)/www/turris-webapps/static/media
 endef
 
 define Package/turris-webapps/postinst
-- 
2.34.1

